media.css

```diff
@@ -0,0 +1,364 @@
+    background-size: cover;
+    margin-top: -50px;
+    height: 600px;
+  }
+
+  .intro::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-image: url(./img/background1.avif);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .intro1 {
+    font-size: 15px;
+    border: 1px solid #fff;
+    width: 610px;
+    margin-left: 190px;
+  }
+
+  .intro-date {
+    font-size: 25px;
+    margin-left: 190px;
+  }
+
+  .intro-location {
+    font-size: 20px;
+    margin-left: 190px;
+    margin-top: 15px;
+  }
+
+  .title1 {
+    font-size: 25px;
+  }
+
+  .hr1 {
+    width: 50px;
+  }
+
+  .info-list {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .info-list .li1 {
+    margin-top: 80px;
+    margin-left: 40px;
+    width: 200px;
+    display: flex;
+    flex-direction: column;
+    height: 150px;
+  }
+
+  .info-list .li2 {
+    margin-top: 80px;
+    margin-left: 40px;
+    width: 200px;
+    display: flex;
+    flex-direction: column;
+    height: 150px;
+  }
+
+  .info-list .li3 {
+    margin-top: 80px;
+    margin-left: 40px;
+    width: 200px;
+    display: flex;
+    flex-direction: column;
+    height: 150px;
+  }
+
+  .info-list .li4 {
+    margin-top: 80px;
+    margin-left: 40px;
+    width: 200px;
+    height: 150px;
+  }
+
+  .info-list .li5 {
+    margin-top: 80px;
+    margin-left: 40px;
+    width: 200px;
+    display: flex;
+    flex-direction: column;
+    height: 150px;
+  }
+
+  .info {
+    margin-top: 10px;
+    margin-left: 0;
+    width: 190px;
+    font-size: 19px;
+  }
+
+  .list1 {
+    text-align: center;
+    font-size: 16px;
+  }
+
+  .btn1 {
+    display: none;
+  }
+
+  .btn2 {
+    display: block;
+    width: 300px;
+    height: 40px;
+    margin: 0 auto;
+    margin-top: 150px;
+    color: #ec5342;
+  }
+
+  .btn2:hover {
+    font-size: 20px;
+  }
+
+  .hr2 {
+    margin-left: 600px;
+    width: 80px;
+    color: #ec5342;
+  }
+
+  .load-more {
+    display: none;
+  }
+
+  .partner {
+    display: none;
+  }
+
+  .partner1 {
+    display: block;
+    border: 1px solid #272a31;
+    background-color: #272a31;
+  }
+
+  .partners2 {
+    margin-top: 10px;
+    font-family: 'Lato', sans-serif;
+    font-size: 15px;
+    text-align: center;
+    color: #d3d3d3;
+  }
+
+  .hr6 {
+    margin-top: 10px;
+    width: 35px;
+    margin-left: 625px;
+    color: #ec5342;
+    margin-bottom: 40px;
+  }
+
+  .partner-list1 {
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+  }
+
+  .partner-list1 li {
+    text-align: center;
+    list-style: none;
+    color: #b3b3b3;
+    font-size: 25px;
+    font-family: 'Cocogoose Pro', sans-serif;
+  }
+
+  .about1 a:hover {
+    color: #ec5342;
+  }
+
+  .title4 {
+    display: none;
+  }
+
+  .title5 {
+    font-size: 50px;
+    width: 800px;
+    padding: 70px;
+    margin-left: 250px;
+  }
+
+  .intro3 {
+    width: 800px;
+    font-size: 25px;
+  }
```

Comment:
- [ ] Kindly make sure your design has the appropriate color and padding 🙏 

Original Desktop Design | Your Desktop Design (About Page)
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/181596111-305ed695-34af-4262-96d5-f9b2aeba25a3.png)  |  ![image](https://user-images.githubusercontent.com/74814780/181596216-31645d20-acbd-47b1-8153-6d9f4bebdc54.png)